### PR TITLE
Release 0.2.3

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
We wish to make a new release.

# Changes
Bump the version number from 0.2.3 to 0.2.3

# Tests
Integration tests:
- Verified that nns-dapp unit tests pass with this new version of the code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns",
-  "version": "0.2.2",
```
Comment (lmuntanerDfinity) [inline package.json:3]: Better to 0.3.0, and check the CHANGELOG.md

We did breaking changes, therefore it makes sense to have a minor upgrade

Comment (bitdivine) [reply package.json:3]: I believe that this PR should be closed in favour of https://github.com/dfinity/nns-js/pull/72 and the 0.2.3 release yanked.  Does that sound OK to you?

Comment (lmuntanerDfinity) [reply package.json:3]: Yes, I agree

**Comment by peterpeterparker** [comment]
We are released v0.3.0 with #72.

**Pull request closed without merging**